src/runner/Resources.resx

```diff
@@ -115,14 +115,14 @@
     <comment>Don't localize "\t" as that is what separates the click portion to be right aligned in the menu.</comment>
   </data>
   <data name="DOCUMENTATION_MENU_TEXT" xml:space="preserve">
-    <value>Documentation</value>
+    <value>Online documentation</value>
   </data>
   <data name="EXIT_MENU_TEXT" xml:space="preserve">
-    <value>Exit</value>
-    <comment>Exit as a verb, as in Exit the application</comment>
+    <value>Close</value>
```

Comment:
Exit is the correct term IMO.  You close windows, you exit the app.

Here is VS
![image](https://github.com/microsoft/PowerToys/assets/1462282/501679ac-c397-49f0-9243-732131d7bdfa)

Here is notepad
![image](https://github.com/microsoft/PowerToys/assets/1462282/a5dfd880-a596-4553-bb37-897287b7d5f8)

Reply:
please revert this one

Reply:
Done